deploy/kube-templates/glusterfs-daemonset.yaml

```diff
@@ -23,6 +23,12 @@ spec:
       - image: gluster/gluster-centos:latest
         imagePullPolicy: IfNotPresent
         name: glusterfs
+        env:
+        - name: SYSTEMD_IGNORE_CHROOT
+          value: "1"
+        command:
+        - /usr/lib/systemd/systemd
+        - --system
```

Comment:
This is NOT recommended from `systemd` in production mode.  so, please dont merge.

Reply:
so, we actually officially supported running systemd as non-PID1 once upon a time, but removed that pretty early on due to bitrot and not having a clear usecase. Supporting that isn't particularly hard to do, running systemd as PID 2 should mostly work (except for C-A-D handling/SIGPWR and things like that) in theory. However, the code is sprinkled with "getpid() == 1" checks to distuingish the modes of operation. If there's a clear usecase, a clear user and somebody stepping up to do the work we could restore this in systemd and officially support it. If so, please file an issue/PR on the system github project. But other than that: it's unlikely to work properly if you run systemd as PID 2 but insist on system mode right now.

Lennart

Reply:
@poettering Thanks for clarification, Lennart. This makes a lot of sense. We'll discuss and possibly come up with a PR/issue/further discussion. 
- Michael

Reply:
@poettering Ah, good to know. Thanks! :)